utils/test/utils.go

```diff
@@ -51,13 +51,20 @@ func FailHandler(t *testing.T) {
 	})
 }
 
-// ServerToClientConfig is used to create client configuration from existing server(s).
-func ServerToClientConfig(servers ...*connection.ServerConfig) *connection.ClientConfig {
+var (
+	// DefaultTLSConfig defines an empty tls config.
+	DefaultTLSConfig connection.TLSConfig
```

Comment:
minor: It was unclear why this PR changes tests from insecure mode to TLS. Renaming this config can make it clearer that the default is insecure.
Furthermore, for most use cases, you can re-introduce `NewInsecureClientConfig()` which was convenient and makes the intent clearer.
```suggestion
	InsecureTLSConfig connection.TLSConfig
```

Reply:
Fixed.